src/storage/src/source/sql_server/replication.rs

```diff
@@ -221,22 +222,44 @@ pub(crate) fn render<G: Scope<Timestamp = Lsn>>(
 
             // Start replicating from the LSN __after__ we took the snapshot.
             let replication_start_lsn = snapshot_lsn.increment();
+            let mut rewinds:BTreeMap<_,_> = export_ids_to_snapshot
```

Comment:
```suggestion
            let mut rewinds: BTreeMap<_, _> = export_ids_to_snapshot
```
rustfmt gives up in this file :(